src/common/Makefile.am

```diff
@@ -103,10 +99,8 @@ libcommon_internal_la_SOURCES += \
 endif
 
 # used by RBD and FileStore
-if LINUX
```

Comment:
probably because it is used by FileStore and BlueStore. can we guard it with `WITH_OSD` and `!SOLARIS?